helm/templates/deployment.yaml

```diff
@@ -55,6 +55,8 @@ spec:
         - "$(ACK_RESOURCE_TAGS)"
         - --watch-namespace
         - "$(ACK_WATCH_NAMESPACE)"
+        - --deletion-policy
+        - "$(DELETION_POLICY)"
```

Comment:
You might need to rebase once https://github.com/aws-controllers-k8s/memorydb-controller/pull/48 is merged

Reply:
Did rebase.